components/nav/index.styl

```diff
@@ -0,0 +1,29 @@
+.navigation
+  display flex
+  width 100%
+  padding 10px
+  avant-garde-size('m-headline')
+  justify-content space-between
+  align-items center
+  border-bottom 1px solid rgba(white, 0.5)
+
+.artsy-logo
+  display inline-block
+  height 30px
+  > svg
+    fill white
+    height 100%
+    vertical-align bottom
+
+.nav-items
+  //
+.nav-item
+  display inline-block
+  text-decoration none
+  padding 2px
+  margin 0 1em
+  &
+  &:visited
+    color white
+  &.is-active
+    border-bottom 2px solid white
```

Comment:
Could toggle this class by checking for which language is active when you re-render